assets/app/views/modals/update-resource.html

```diff
@@ -0,0 +1,26 @@
+<div class="modal-resource-delete">
+  <div class="modal-body">
+    <div ng-if="resourceList.length === 1">
+      <h1>{{resourceKind | humanize}} '<strong>{{resourceName}}</strong>' already exists</h1>
+      <p>Do you want to replace the current '<strong>{{resourceName}}</strong>' {{resourceKind | humanize}} with the submitted one?</p>
+    </div>
+    
+    <div ng-if="resourceList.length > 1">
+      <h1>Some of {{resourceKind | humanize}} items already exists</h1>
```

Comment:
There can be only one top level `Kind`, in this particular case it will be List. Listed will be only those items from the List that already exist, whatever type they are.